includes/polling/ports.inc.php

```diff
@@ -936,12 +954,21 @@
 
         // Update Database
         if (count($port['update'])) {
-            $updated = dbUpdate($port['update'], 'ports', '`port_id` = ?', array($port_id));
+            if (!empty($port['update'])) {
+                $port['update']['poll_time'] = $polled;
+                $port['update']['poll_prev'] = $port['poll_time'];
+                $port['update']['poll_period'] = $polled_period;
+                $updated = dbUpdate($port['update'], 'ports', '`port_id` = ?', array($port_id));
+            } else {
+                $port_ids[] =  $port_id;
```

Comment:
`if (!empty($port['update'])`  is pretty much the same as `if (count($port['update']))` so this will always be true  (I do prefer the not empty notation)

This means this else is a dead code path.